frontend/src/components/Bookshelf/Bookshelf.jsx

```diff
@@ -189,7 +224,9 @@ const Bookshelf = () => {
     console.log('active index: ' + activeIndex);
     console.log('over index: ' + overIndex);
 
-    // if the draggable is dragged into a new position
+    const activeIndex = items[activeContainer].findIndex(book => book.cover === active.id);
```

Comment:
i think lines 219, 220, 224, 225 need to be deleted for this commit to be functional!
as 227 and 228 define activeIndex properly.